morgoth/addons/models.py

```diff
@@ -78,12 +78,16 @@ def __repr__(self):
         return '<VersionNumber: {}>'.format(self)
 
     def __eq__(self, other):
-        if not isinstance(other, VersionNumber):
+        if other is None:
```

Comment:
When is `other` `None`? During a test?

I feel like comparing it to `None` should throw too, and wherever it's currently happening should either catch or otherwise change. But I could easily be convinced otherwise.